### PR TITLE
Fix UI issues#4726

### PR DESCRIPTION
**Pull request by lovelgeorge99**
Issue #4726 
made changes for 1,2,3,4,6 
i am still working on 5

## Summary by CodeRabbit

- **New Features**
	- Adjusted text display for the ShareLikeBadge based on donation status.
	- Simplified layout and spacing for various project components, enhancing visual presentation.

- **Bug Fixes**
	- Improved icon visibility in ProjectPublicActions based on user interaction.

- **Style**
	- Updated height and spacing properties for ProjectHeader and other components for better alignment and aesthetics.
	- Modified margin and padding properties in multiple components to improve layout consistency.

### CODE REVIEW

File: src/components/views/project/projectActionCard/ProjectActionCard.tsx [hunk 1]
```diff
@@ -57,20 +55,7 @@ const ProjectActionInnerCard = () => {
...
-				>
-					<Subline>
-						{formatMessage({
-							id: 'label.learn_more_about_donating_on_giveth',
-						})}
-					</Subline>
-					<IconChevronRight16 />
-				</LearnLink>
-			</Link>
+			<Link href={Routes.Onboarding + '/donors'}></Link>
```
[the remark this answers is not in the document]
Comment (kkatusic) [reply src/components/views/project/projectActionCard/ProjectActionCard.tsx]: @lovelgeorge99 can you check why this link is empty? thx

[the remark this answers is not in the document]
Comment (lovelgeorge99) [reply src/components/views/project/projectActionCard/ProjectActionCard.tsx]: Yes i removed the empty link on later commit.

**Review by kkatusic — APPROVED** [review]
LGTM ;) thx @lovelgeorge99

**Review by MohammadPCh — APPROVED** [review]
Thanks @lovelgeorge99 LGTM

**Pull request merged**